streams/src/test/java/org/apache/kafka/streams/processor/internals/DefaultStateUpdaterTest.java

```diff
@@ -498,6 +575,66 @@ private void shouldNotRemoveTaskFromFailedTasks(final Task task) throws Exceptio
         verifyRestoredActiveTasks();
     }
 
+    @Test
+    public void shouldNotPauseActiveStatefulTaskFromRestoredActiveTasks() throws Exception {
```

Comment:
`shouldNotPauseActiveStatefulTaskFromRestoredActiveTasks` -> `shouldNotPauseActiveStatefulTaskInRestoredActiveTasks`

The same applies to below test methods.

Reply:
Ack.